chart/tests/test_flower.py

```diff
@@ -22,12 +22,21 @@
 from tests.helm_template_generator import render_chart
 
 
-class FlowerAuthorizationTest(unittest.TestCase):
+class FlowerTest(unittest.TestCase):
+    def test_should_create_flower_by_celery_executor(self):
+        docs = render_chart(
+            values={"executor": "CeleryExecutor", "flower": {"enabled": True}},
```

Comment:
you think it is worth parameterizing to confim that it is created if and only if enabled?